block_function.rb

```diff
@@ -0,0 +1,14 @@
+# Create a program block_function.rb.  It should have a function do_calc that calls a
+# block using a yield statement.  The yield statement will pass the numbers 7 and
+# 9 to a block, and then will print out the result.  Call the do_calc function twice in your program. The first time, pass a block that adds the two numbers. The second time,
+# pass a block that multiplies the two numbers.  Your program should print out 16 and 63.   
+
+
+def do_calc # yield just acts as a blind return? what is this for? for classes i assume?
```

Comment:
The yield allows for you to pass behavior to a method  to be dynamic/flexible. For example when we call `do_calc` we can tell it what we want to do subtraction, division, multiplication, addition but the input numbers will always be the same because we defined it in the `yield(7,9)`